hack/local-up-cluster.sh

```diff
@@ -539,7 +539,7 @@ function start_apiserver {
     fi
 
     APISERVER_LOG=${LOG_DIR}/kube-apiserver.log
-    ${CONTROLPLANE_SUDO} "${GO_OUT}/hyperkube" apiserver ${swagger_arg} ${audit_arg} ${authorizer_arg} ${priv_arg} ${runtime_config} \
+    ${CONTROLPLANE_SUDO} "${GO_OUT}/hyperkube" kube-apiserver ${swagger_arg} ${audit_arg} ${authorizer_arg} ${priv_arg} ${runtime_config} \
```

Comment:
? Isn't this a breaking change?  Or is the old one deprecated?

Reply:
> ? Isn't this a breaking change? Or is the old one deprecated?

Gotta be a bug from 1.0, right?  I can re-alias it if need be.